app/views/shopify_app/sessions/enable_cookies.html.erb

```diff
@@ -2,350 +2,20 @@
-
-    .Polaris-Button:focus {
-      border-color:#5c6ac4;
-      outline:0;
-      box-shadow:0 0 0 1px #5c6ac4;
-    }
-
-    .Polaris-Button:active {
-      background:linear-gradient(to bottom, #f4f6f8, #f4f6f8);
-      border-color:#c4cdd5;
-      box-shadow:0 0 0 0 transparent, inset 0 1px 1px 0 rgba(99, 115, 129, 0.1), inset 0 1px 4px 0 rgba(99, 115, 129, 0.2);
-    }
-
-    .Polaris-Button.Polaris-Button--disabled {
-      fill:#919eab;
-      transition:none;
-      background:linear-gradient(to bottom, #f4f6f8, #f4f6f8);
-      color:#919eab;
-    }
-
-    .Polaris-Button__Content {
-      font-size:1.5rem;
-      font-weight:400;
-      line-height:1.6rem;
-      text-transform:initial;
-      letter-spacing:initial;
-      position:relative;
-      display:-webkit-box;
-      display:-ms-flexbox;
-      display:flex;
-      -webkit-box-pack:center;
-      -ms-flex-pack:center;
-      justify-content:center;
-      -webkit-box-align:center;
-      -ms-flex-align:center;
-      align-items:center;
-      min-width:1px;
-      min-height:1px;
-    }
-      
-    @media (min-width: 40em) {
-      .Polaris-Button__Content {
-        font-size:1.4rem;
-      } 
-    }
-
-    .Polaris-Button--primary {
-      background:linear-gradient(to bottom, #6371c7, #5563c1);
-      border-color:#3f4eae;
-      box-shadow:inset 0 1px 0 0 #6774c8, 0 1px 0 0 rgba(22, 29, 37, 0.05), 0 0 0 0 transparent;
-      color:white;
-      fill:white;
-    }
-
-    .Polaris-Button--primary:hover {
-      background:linear-gradient(to bottom, #5c6ac4, #4959bd);
-      border-color:#3f4eae;
-      color:white;
-      text-decoration:none;
-    }
-
-    .Polaris-Button--primary:focus {
-      border-color:#202e78;
-      box-shadow:inset 0 1px 0 0 #6f7bcb, 0 1px 0 0 rgba(22, 29, 37, 0.05), 0 0 0 1px #202e78;
-    }
-
-    .Polaris-Button--primary:active {
-      background:linear-gradient(to bottom, #3f4eae, #3f4eae);
-      border-color:#38469b;
-      box-shadow:inset 0 0 0 0 transparent, 0 1px 0 0 rgba(22, 29, 37, 0.05), 0 0 1px 0 #38469b;
-    }
-
-    .Polaris-Button--primary.Polaris-Button--disabled {
-      fill:white;
-      background:linear-gradient(to bottom, #bac0e6, #bac0e6);
-      border-color:#a7aedf;
-      box-shadow:none;
-      color:white;
-    }
-  </style>
   <base target="_top">
   <title>Redirecting…</title>
 
+  <%= stylesheet_link_tag('styles') %>
+  <%= stylesheet_link_tag('itp') %>
+
   <script>
     window.apiKey = "<%= ShopifyApp.configuration.api_key %>";
     window.shopOrigin = "https://<%= @shop %>";
   </script>
 
-  <%= javascript_include_tag('shopify_app/itp_polyfill', crossorigin: 'anonymous', integrity: true) %>
+  <%= javascript_include_tag('shopify_app/itp_helper', crossorigin: 'anonymous', integrity: true) %>
+  <%= javascript_include_tag('shopify_app/storage_access', crossorigin: 'anonymous', integrity: true) %>
+  <%= javascript_include_tag('shopify_app/partition_cookies', crossorigin: 'anonymous', integrity: true) %>
```

Comment:
similar question to css stuff above — can we combine these into a single bundle to improve performance?

Reply:
@tylerball Since we're using Webpack for our Karma tests, should we use it to bundle our JS as well?

Reply:
How would that work in practice, since this is a gem? Would the app developer need to set up their environments to run webpack in dev/prod?

Reply:
Mike is right, we need to keep these as sprockets includes because this is a gem, but we could use `//= require`s inside of the scripts to only import a single JS file.